packages/cosmos_ui_components/lib/components/template/cosmos_wallet_list_item.dart

```diff
@@ -1,53 +1,67 @@
+import 'package:cosmos_ui_components/components/cosmos_wallet_edit_button.dart';
 import 'package:cosmos_ui_components/components/gradient_avatar.dart';
+import 'package:cosmos_ui_components/cosmos_text_theme.dart';
 import 'package:cosmos_ui_components/cosmos_theme.dart';
 import 'package:flutter/material.dart';
 
 class CosmosWalletListItem extends StatelessWidget {
   final String name;
   final String address;
   final bool isSelected;
+  final bool isEditing;
```

Comment:
whe adding new parameters to components, lets refrain from marking them as `required` but instead provide the default value to make it as much backward-compatibile as possible

Reply:
Updated